pyk4a/pyk4a.cpp

```diff
@@ -192,6 +192,53 @@ extern "C" {
         }
     }
 
+	
+	    static PyObject* device_get_ir_image(PyObject* self, PyObject* args){
```

Comment:
Getting the ir image this way is cleaner.
We could get it during with the depth instead. It may be possible to transform the depth and the ir at the same time for speed purposes. This is fine for now.